client/cmd/kcoin/main.go

```diff
@@ -254,6 +256,11 @@ func startNode(ctx *cli.Context, stack *node.Node) {
 	debug.Memsize.Add("node", stack)
 	setupLogging(ctx)
 
+	// make use client runs latest major version if not in testnet mode, need to check pre Start node!
+	if !ctx.GlobalBool(utils.TestnetFlag.Name) {
+		mustBeLatestMajorVersion(ctx)
```

Comment:
help, this method name is odd

Reply:
That seems idiomatic, it's just necessarily long.